### DISCUSSION TITLE
Add the osu!stream skin as a option

### DISCUSSION THREAD

**Discussion by letsgoawaydev** [ppy/osu#23356] [Ideas] [answered]
This probably isnt the greatest idea, but i thought it would make sense considering on iOS the only official osu! game on the app store is osu!stream, and maybe it could help people convert over to lazer? just an idea i had

edit: and also because i heard lazer on mobile is going to be improved on soon, so this could improve it maybe

**Comment by peppy** [comment]
Might happen one day. I wouldn't say no if someone implemented this.

**Reply by letsgoawaydev** [reply]
cool! i might try to make it and see how it goes

**Reply by letsgoawaydev** [reply]
just realised there are a few issues with this 

- the absence of modes in the skin like mania catch and taiko
- the absence of a cursor (the cursor is only particles in osu!stream)

**Comment by letsgoawaydev** [comment, marked as answer]
Made a PR for this: https://github.com/ppy/osu/pull/35206
